<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelowanieGeometryczne1Class</class>
 <widget class="QMainWindow" name="ModelowanieGeometryczne1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModelowanieGeometryczne1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0" rowspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="MyGLWidget" name="myGLWidget">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>600</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80000</width>
            <height>60000</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="myTabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_Objects">
           <attribute name="title">
            <string>Objects</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="ListWidgetObjects" name="listWidget_ObjectsList">
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButton_AddObject">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_DeleteObject">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_DrawableObjects">
              <property name="modelColumn">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_Parameters">
           <attribute name="title">
            <string>Parameters</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolBox" name="toolBox">
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <property name="tabSpacing">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="page_Torus">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>293</width>
                   <height>505</height>
                  </rect>
                 </property>
                 <attribute name="label">
                  <string>Torus</string>
                 </attribute>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="ComboBoxTorus" name="comboBox_Torus"/>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>Small radius</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDoubleSpinBox" name="doubleSpinBox_smallR">
                      <property name="minimum">
                       <double>0.050000000000000</double>
                      </property>
                      <property name="maximum">
                       <double>1.000000000000000</double>
                      </property>
                      <property name="singleStep">
                       <double>0.050000000000000</double>
                      </property>
                      <property name="value">
                       <double>0.100000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>Big radius</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDoubleSpinBox" name="doubleSpinBox_bigR">
                      <property name="maximum">
                       <double>4.000000000000000</double>
                      </property>
                      <property name="singleStep">
                       <double>0.250000000000000</double>
                      </property>
                      <property name="value">
                       <double>0.300000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Minor segments</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="spinBox_minorSegments">
                      <property name="minimum">
                       <number>1</number>
                      </property>
                      <property name="maximum">
                       <number>100</number>
                      </property>
                      <property name="value">
                       <number>30</number>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>Major segments</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="spinBox_majorSegments">
                      <property name="minimum">
                       <number>1</number>
                      </property>
                      <property name="maximum">
                       <number>100</number>
                      </property>
                      <property name="value">
                       <number>30</number>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="verticalSpacer">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="page_BezierCurveC0">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>293</width>
                   <height>505</height>
                  </rect>
                 </property>
                 <attribute name="label">
                  <string>Bezier Curve C0</string>
                 </attribute>
                 <layout class="QVBoxLayout" name="verticalLayout_11">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,0">
                    <property name="spacing">
                     <number>5</number>
                    </property>
                    <item>
                     <widget class="ComboBoxBezierCurveC0" name="comboBox_BezierCurveC0"/>
                    </item>
                    <item>
                     <widget class="ListWidgetParameters" name="listWidget_Parameters"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_Other">
           <attribute name="title">
            <string>Other features</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="radioButton_stereo">
                <property name="text">
                 <string>Stereoscopy on/off</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_stereoscopy_slider">
                <property name="text">
                 <string>Stereoscopy e parameter:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox_e">
                <property name="minimum">
                 <double>-1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
                <property name="value">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignVCenter">
           <widget class="QCheckBox" name="checkBox_pointer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>10</height>
             </size>
            </property>
            <property name="text">
             <string>3d pointer</string>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_screenCoords">
            <property name="text">
             <string>Screen coords:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3dCoords">
            <property name="text">
             <string>Scene coords:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>26</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">myglwidget.h</header>
  </customwidget>
  <customwidget>
   <class>ListWidgetObjects</class>
   <extends>QListWidget</extends>
   <header location="global">listwidgetobjects.h</header>
  </customwidget>
  <customwidget>
   <class>ListWidgetParameters</class>
   <extends>QListWidget</extends>
   <header location="global">listwidgetparameters.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBoxBezierCurveC0</class>
   <extends>QComboBox</extends>
   <header location="global">comboboxBezierCurveC0.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBoxTorus</class>
   <extends>QComboBox</extends>
   <header location="global">comboboxTorus.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="modelowaniegeometryczne1.qrc"/>
 </resources>
 <connections/>
</ui>
